pkg/controller/clusterautoscaler/clusterautoscaler.go

```diff
@@ -34,6 +34,7 @@ func (a AutoscalerArg) TypeRange(t string, min, max int) string {
 // operator when processing ClusterAutoscaler resources.
 const (
 	LogToStderrArg                  AutoscalerArg = "--logtostderr"
+	VerboseArg                      AutoscalerArg = "--v=3"
```

Comment:
This should not have a value. It should just be `--v` and then `VerboseArg.Value("3")` can be used to get this level. I would actually prefer a `CLUSTER_AUTOSCALER_VERBOSITY` setting that set this, but we don't have to do that now.